### ISSUE TITLE
Why not add more performance mods

### ISSUE THREAD

**Issue by DarkStas** [Divine-Journey-2/Divine-Journey-2#1021]
### Mod Name

performant-1.11, phosphor-0.2.7, Nothirium-0.3.2, censoredasm5.14

### Reasons why this should be added

More performance, faster chunk load, less memory use (checked on solo-server). I tried it on my old solo server, for tier 5 it had no issues (3 days ago when I played game after tier 5 it started to drop fps from 120 to 80-90 fps so i migrated to online server to have my 120 fps back (that helped), so again, I went to self-created online server. But sadly, after crafting railgun and have it in hand, server perf dropped by 2 times (without this mods and with them), from 120 to 60...). 

But it's better to try on full tier explored servers. 

Will add links to mods, "+" mean that essential addon to higher up mod, "-" mean that this mod need to be deleted

performant - https://legacy.curseforge.com/minecraft/mc-mods/performant
phosphor - https://legacy.curseforge.com/minecraft/mc-mods/phosphor-forge
"+" PhosphorCrashFix-1.12.2-1.0.0 - https://legacy.curseforge.com/minecraft/mc-mods/phosphorcrashfix/files
Nothirium - https://www.curseforge.com/minecraft/mc-mods/nothirium
"+" [___MixinCompat-1.1-1.12.2___] - https://legacy.curseforge.com/minecraft/mc-mods/mixin-0-7-0-8-compatibility
"+" _MixinBootstrap-1.1.0 - https://legacy.curseforge.com/minecraft/mc-mods/mixinbootstrap
"+" RenderLib-1.12.2-1.3.1 - https://legacy.curseforge.com/minecraft/mc-mods/renderlib
censoredasm - https://www.curseforge.com/minecraft/mc-mods/lolasm
"-" spark 1.6.3
"+" spark 1.5.2 - https://www.curseforge.com/minecraft/mc-mods/spark
"-" VanillaFix

**Comment by DarkStas** [comment]
Finded why my fps dropped at first to 90 fps and then to 60 fps. Its all thanks to "ImmersiveEngineering-0.12-98" and railgun (fps drop from 120 to 60-70 fps) and mining drill (fps drop from 120 to 90 fps) with both of them its 50-60 fps...
Will write bug report...

**Comment by DarkStas** [comment]
Tried AiReducer but it cause mobs to lag/freeze for 1-3 sec, and then make a all movements in next 1 sec after unfreeze...

Find some more performance mods, will try them too. 

Will add EntityCulling to upper list, it give some performance boost and work fine with mods until tier 6.

**Comment by Atricos** [comment]
Can you explain why Spark 1.5.2 is better than Spark 1.6.3?

**Comment by DarkStas** [comment]
> 

censoredasm work only with this version. 
All "+" and "-" based on mod requirements.

**Comment by Irgendwer01** [comment]
> ### Mod Name
> 
> performant-1.11, phosphor-0.2.7, Nothirium-0.3.2, censoredasm5.14
> ### Reasons why this should be added
> 
> More performance, faster chunk load, less memory use (checked on solo-server). I tried it on my old solo server, for tier 5 it had no issues (3 days ago when I played game after tier 5 it started to drop fps from 120 to 80-90 fps so i migrated to online server to have my 120 fps back (that helped), so again, I went to self-created online server. But sadly, after crafting railgun and have it in hand, server perf dropped by 2 times (without this mods and with them), from 120 to 60...).
> 
> But it's better to try on full tier explored servers.
> 
> Will add links to mods, "+" mean that essential addon to higher up mod, "-" mean that this mod need to be deleted
> 
> performant - https://legacy.curseforge.com/minecraft/mc-mods/performant phosphor - https://legacy.curseforge.com/minecraft/mc-mods/phosphor-forge "+" PhosphorCrashFix-1.12.2-1.0.0 - https://legacy.curseforge.com/minecraft/mc-mods/phosphorcrashfix/files Nothirium - https://www.curseforge.com/minecraft/mc-mods/nothirium "+" [_**MixinCompat-1.1-1.12.2**_] - https://legacy.curseforge.com/minecraft/mc-mods/mixin-0-7-0-8-compatibility "+" _MixinBootstrap-1.1.0 - https://legacy.curseforge.com/minecraft/mc-mods/mixinbootstrap "+" RenderLib-1.12.2-1.3.1 - https://legacy.curseforge.com/minecraft/mc-mods/renderlib censoredasm - https://www.curseforge.com/minecraft/mc-mods/lolasm "-" spark 1.6.3 "+" spark 1.5.2 - https://www.curseforge.com/minecraft/mc-mods/spark "-" VanillaFix
> ### Additional context
> 
> _No response_
> ### Your Divine Journey 2 Discord Username
> 
> _No response_

I'd suggest to not use Performant and Nothirium as both rather cause issues.
Phosphor should be replaced through https://legacy.curseforge.com/minecraft/mc-mods/hesperus as it's an continued fork of it.
Else the list looks good.

**Comment by DarkStas** [comment]
How I said, I tested with all mods that I suggested (and Performant and Nothirium) and the only bug is that time to time mobs freeze (I think its performant(it stop mobs tick for server to unlag from fast movement and chunk load/generation), before this thought that it was AiReducer and EntityCulling, but problem reoccur time to time without them, so need to be tested without performant but with AiReducer and EntityCulling, EntityCulling give massive boost in fps by render only where you looking, not 360. On my tier 8 quest line I got from 100 fps back to cap of 120). 

Also, with all this mods, I got massive chunk load speed increase. And loading speed to 8-9 min (was 11-12m).
